sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/KeyVaultEnvironmentPostProcessorHelper.java

```diff
@@ -79,6 +81,7 @@ public void addKeyVaultPropertySource(String normalizedName) {
         final SecretClient secretClient = new SecretClientBuilder()
                 .vaultUrl(vaultUri)
                 .credential(tokenCredential)
+                .serviceVersion(SecretServiceVersion.V7_0)
```

Comment:
Why do we need this value here?

Reply:
by default, serviceVersion value is V7_1, but the starter run failed with errors below
```
{"error":{"code":"BadParameter","message":"The specified version (7.1) is not recognized. Consider using the latest supported version (2016-10-01)."}}
```